example/project-rails-tailwind/test/components/stories/button_component_stories.stories.json

```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Atoms",
+  "stories": [
+    {
+      "name": "button",
```

Comment:
nit for consistency across example repos:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/82983755/227344571-d8120768-2ad3-4aa4-aa76-dfb41a8a98d9.png">
Capitalizing where it says `button`, `input`, etc. Not sure if that change is made in the `component.stories.json` file or in the `component_stories.rb` file.

Reply:
I think it's rails convention to keep it lower cased.